CONTRIBUTING.rst

```diff
@@ -123,20 +155,6 @@ Alternatively, you can generate the necessary files in ``mlflow/server/js/build`
 `Building a Distributable Artifact`_.
 
 
-Tests and Lint
---------------
-Verify that the unit tests & linter pass before submitting a pull request by running:
-
-.. code-block:: bash
-
-    pytest
-    ./lint.sh
-
-When running ``pytest --requires-ssh`` it is necessary that passwordless SSH access to localhost
```

Comment:
Is this no longer relevant?

Reply:
Ah no it still is, I'll add it back + clarify that some tests aren't run in Travis by default (& how to run them).